addons/test_mail/tests/test_mail_activity_mixin.py

```diff
@@ -126,6 +126,34 @@ def test_activity_mixin(self):
             self.assertFalse(self.test_record.activity_state)
             self.assertFalse(act2.exists())
 
+    @users('employee')
+    def test_activity_attachments_ids(self):
```

Comment:
Not related to activities, this adds noise when trying to reduce / remove / rewrite tests (and is probably going to be removed at some point). Have dedicated tests for ORM issues is better ^^

Reply:
Agreed, the test should be in module `test_orm`.